src/bp-activity/bp-activity-filters.php

```diff
@@ -2360,25 +2043,489 @@ function bp_blogs_activity_content_with_read_more( $content, $activity ) {
  * @since BuddyBoss 1.5.5
  */
 function bp_blogs_activity_comment_content_with_read_more( $content, $activity ) {
-
-	if( 'activity_comment' === $activity->type && $activity->item_id && $activity->item_id > 0 ) {
+	if ( 'activity_comment' === $activity->type && $activity->item_id && $activity->item_id > 0 ) {
 		// Get activity object.
 		$comment_activity = new BP_Activity_Activity( $activity->item_id );
-		if( 'blogs' === $comment_activity->component && isset( $comment_activity->secondary_item_id ) && 'new_blog_' . get_post_type( $comment_activity->secondary_item_id ) === $comment_activity->type ) {
+		if ( 'blogs' === $comment_activity->component && isset( $comment_activity->secondary_item_id ) && 'new_blog_' . get_post_type( $comment_activity->secondary_item_id ) === $comment_activity->type ) {
 			$comment_post_type = $comment_activity->secondary_item_id;
 			$get_post_type     = get_post_type( $comment_post_type );
 			$comment_id        = bp_activity_get_meta( $activity->id, 'bp_blogs_' . $get_post_type . '_comment_id', true );
-			if( $comment_id ) {
+			if ( $comment_id ) {
 				$comment = get_comment( $comment_id );
 				$content = bp_create_excerpt( html_entity_decode( $comment->comment_content ) );
-				if( false !== strrpos( $content, __( '&hellip;', 'buddyboss' ) ) ) {
+				if ( false !== strrpos( $content, __( '&hellip;', 'buddyboss' ) ) ) {
 					$content     = str_replace( ' [&hellip;]', '&hellip;', $content );
 					$append_text = apply_filters( 'bp_activity_excerpt_append_text', __( ' Read more', 'buddyboss' ) );
 					$content     = sprintf( '%1$s<span class="activity-blog-post-link"><a href="%2$s" rel="nofollow">%3$s</a></span>', $content, get_comment_link( $comment_id ), $append_text );
 				}
 			}
 		}
 	}
-
 	return $content;
 }
+/**
+ * Media comment migration for activity.
+ *
+ * @param array $repair_list List of all repair items.
+ *
+ * @return array Repair list items.
+ */
+function bb_activity_media_document_migration( $repair_list ) {
+	$repair_list[] = array(
+		'bp-repair-activity-media',
+		__( 'Repair Media Comments on the Activity/News Feed.', 'buddyboss' ),
+		'bb_activity_media_document_migration_process',
+	);
+	return $repair_list;
+}
+/**
+ * Activity media migration process.
+ */
+function bb_activity_media_document_migration_process() {
+	global $wpdb;
+	$bp                  = buddypress();
+	$activity_table_name = $bp->activity->table_name;
+	$get_offset          = filter_input( INPUT_POST, 'offset', FILTER_SANITIZE_NUMBER_INT );
+	$offset              = isset( $get_offset ) ? (int) ( $get_offset ) : 0;
+	if ( 1 === $offset ) {
+		$offset = 0;
+	} else {
+		$offset = $offset;
+	}
+	$get_active_post_types = get_post_types( array( 'public' => true ) );
```

Comment:
can we change the variable name `$get_active_post_types` to `$active_post_types` ? 

Whenever we get data from function which name suggests it is getting data, we just need that data's name not get,fetch,update etc in the variable name.